src/main/java/uk/gov/hmcts/reform/idam/client/models/UserInfo.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.idam.client.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@Builder
+@NoArgsConstructor
```

Comment:
why is this needed?

Reply:
Good spot, no need for that in immutable classes.